frontend/src/metabase/querying/filters/components/FilterPicker/FilterColumnPicker/FilterColumnPicker.tsx

```diff
@@ -36,7 +36,12 @@ type Section = BaseSection<Item> & {
   key?: string;
 };
 
-const SEARCH_PROP = ["name", "displayName", "combinedDisplayName"] as const;
+const SEARCH_PROP = [
```

Comment:
Probably unrelated, but FYI I get this warning when I start typing `[` in the expression editor

![image](https://github.com/user-attachments/assets/8289c77c-407d-4eeb-be32-16fcf56e3d5b)